src/main/resources/jenkins/plugins/hipchat/workflow/HipChatStep/config.jelly

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="message" title="Message">
+        <f:textbox/>
+    </f:entry>
+    <f:entry field="color" title="Color">
```

Comment:
So that works to populate the dropdown - title is string value and value is Enum name. But using ```color: "GREEN"``` or ```color: "green"``` results in:
```
java.lang.IllegalArgumentException: Expected named arguments but got [{color=GREEN}, simple-workflow 93]
```
Anyone know of any Workflow steps with a field backed by an Enum?